src/google_sheets/pyproject.toml

```
@@ -0,0 +1,42 @@
+[project]
+name = "mitol-django-google-sheets"
+version = "2023.12.19.1"
```

A recent release was made:
```suggestion
version = "2024.7.3"
```

Also needs to be updated in this package's `__init__.py`, but github doesn't let me comment on that file because it was only moved.